### DISCUSSION TITLE
Quasar Microfrontend & Native App

### DISCUSSION THREAD

**Discussion by jmriyaz84** [quasarframework/quasar#15443] [CLI - Capacitor / Cordova modes]
Dear Quasar Team,
We are developing hybrid (web & mobile) application for big industrial organization which has 7000+ user. By using 
**Frontend : Quasar 2
Backend : Springboot 
Authentication : Azure Active Directory** 
Application has more than 15 modules & we developing Microservices for each module in backend.
But in frontend we are dumping entire code in single repo & server instead we want to use Microfrontend for each module.
**Now the challenge is how we can handle this microfrontend while converting into Native Mobile Application through Capacitor / Cordova.**
Please share your ideas and thought on this.

**Comment by corepay** [comment]
I don't know about capacitor but have been pretty successful so far using module federation with quasar as a Vue plugin.  Have 7 or 8 apps in context like Customer, Products each with their own forms, settings, tables etc that deploy independently and several client apps pulling in components from each of the context apps as needed.  One app is an app that’s  has all my common/shared components and vuex store and helpers/utilities I import into the context and client apps.
